### PR TITLE
completed react-hash-routing

### PR DESCRIPTION
**Pull request by AndyPark20**
hash routing

### CODE REVIEW

File: react-hash-routing/client/components/header.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default function Header(props) {
+  return (
+    <header className="mb-5">
+      <nav className="navbar navbar-dark bg-dark shadow-sm">
+        <div className="container">
+          <div className="col px-0">
+            {/* this anchor should go back to the catalog at '#' */}
+            <a href="" className="navbar-brand">
```
Comment (Zhiwutian) [inline react-hash-routing/client/components/header.jsx:10]: make sure to update here!

File: react-hash-routing/client/app.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+  componentDidMount() {
+    /**
+     * Listen for hash change events on the window object
+     * Each time the window.location.hash changes, parse
+     * it with the parseRoute() function and update state
+     */
+    window.addEventListener('hashchange',()=>{
+      const changedHash = window.location.hash;
+      console.log('value',changedHash)
```
Comment (Zhiwutian) [inline react-hash-routing/client/app.jsx:24]: console log!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
tiny updates!

File: react-hash-routing/client/app.jsx (hunk 2 above)
Comment (AndyPark20) [reply react-hash-routing/client/app.jsx:24]: ahhh!! roger that!!!

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**